client/my-sites/domains/domain-management/dns/ns-record.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+class NsRecord extends React.Component {
+	static propTypes = {
+		fieldValues: PropTypes.object.isRequired,
+		onChange: PropTypes.func.isRequired,
+		selectedDomainName: PropTypes.string.isRequired,
+		show: PropTypes.bool.isRequired,
+	};
+
+	render() {
+		const { fieldValues, isValid, onChange, selectedDomainName, show, translate } = this.props;
+		const classes = classnames( { 'is-hidden': ! show } );
+		const isNameValid = isValid( 'name' );
+		const isDataValid = isValid( 'data' );
+
+		return (
+			<div className={ classes }>
+				<FormFieldset>
+					<FormLabel>{ translate( 'Name', { context: 'Dns Record' } ) }</FormLabel>
+					<FormTextInputWithAffixes
+						name="name"
+						placeholder={ translate( 'Enter subdomain (required)', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Enter subdomain (required)', { context: 'Placeholder shown when entering the required subdomain part of a new DNS record'} )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/dns-ns-record-editing)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?